data/json/recipes/recipe_others.json

```diff
@@ -5050,7 +5050,39 @@
       [ "silver_small", 50 ]
      ]
     ]
-},{
+},
+{
+  "type" : "recipe",
+  "result": "mirror",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "fabrication",
+  "difficulty": 2,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
```

Comment:
This recipe is completely unavailable to the player, it is neither auto learned, nor learned through disassembling (non-reversible) nor is it in any book.

The other hand mirror recipe can be learned from the glass blowing book. It contains this:
```JSON
  "book_learn": [[ "glassblowing_book", 5 ]],
```
Maybe this should be added to this recipe as well?